File: app/src/main/java/cf/movie/slmovie/main/search/event/SearchEvent.java
package cf.movie.slmovie.main.search.event;

import cf.movie.slmovie.eventBus.BaseEvent;
import cf.movie.slmovie.main.search.bean.SearchAdapter;

/**
 * Created by 包俊 on 2017/8/12.
 */

public class SearchEvent extends BaseEvent {

    private SearchAdapter adapter;

    public SearchEvent(boolean status, String message) {
        super(status, message);
    }

    public SearchAdapter getAdapter() {
        return adapter;
    }

    public void setAdapter(SearchAdapter adapter) {
        this.adapter = adapter;
    }
}

File: app/src/main/java/cf/movie/slmovie/main/search/event/SearchEvent.kt
package cf.movie.slmovie.main.search.event

import cf.movie.slmovie.eventBus.BaseEvent
import cf.movie.slmovie.main.search.bean.SearchAdapter
import cf.movie.slmovie.main.search.bean.SearchResult
import java.util.*

/**
 * Created by 包俊 on 2017/8/12.
 */

class SearchEvent(status: Boolean, message: String) : BaseEvent(status, message) {

    var adapter: SearchAdapter? = null

    var movies: ArrayList<SearchResult.Movies>? = null
}
